### DISCUSSION TITLE
Adding link to Video for Leaderboards

### DISCUSSION THREAD

**Discussion by ghost** [RetroAchievements/RAWeb#765] [Ideas]
A option to adding/editing a link to a video/recorded-stream on the Leaderboards once a Score/Time has been submitted.

Example:
![Video Link on Leaderboards](https://i.imgur.com/ZGDSvKQ.png)

**Comment by Tsearo** [comment]
I'd love to see this and can try to see how much work it would be to implement. It would likely require a new column in `leaderboardentry` for storing/updating the link and then would just need to find a good way to add an option for the user to set/update the link for their own entries. Could possibly have an expandable div that displays the embedded video below the entry when clicked.
